src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -244,16 +245,23 @@ public void setConfig(File file) {
     // -@cs[AbbreviationAsWordInName] Should be removed at 7.0 version,
     // we keep for some time to avoid braking compatibility.
     @Deprecated
-    public void setConfigURL(URL url) {
+    public void setConfigURL(String url) {
         setConfigUrl(url);
     }
 
     /**
      * Sets URL to the configuration.
      * @param url the URL of the configuration to use
      */
-    public void setConfigUrl(URL url) {
-        setConfigLocation(url.toExternalForm());
+    public void setConfigUrl(String url) {
+        try {
+            setConfigLocation(new URL(url).toExternalForm());
```

Comment:
Not clear what you're suggesting - do you want me to use `CommonUtils.getUriByFilename()` instead, or are you saying `setConfigUrl()` should simply be:

```
public void setConfigUrl(String url) {
    setConfigLocation(url);
}
```
?

I think that the latter would be a breaking change for callers that rely on `.toExternalForm()` being called, but if that doesn't concern you I can change it to that.